gatorgrade/output/test_output_functions.py

```diff
@@ -0,0 +1,57 @@
+"""Test suite for output_functions.py."""
```

Comment:
This file should be in the `/tests` folder with the other pytest files! Also it's missing the `import pytest`.

Reply:
Linting detects it as an unused import so I can't use`import pytest`.